restore.py

```diff
@@ -206,6 +206,7 @@ def restore_partitions():
             util.bindMount("/dev", "%s/dev" % dest_fs.mount_point)
             util.bindMount("/sys", "%s/sys" % dest_fs.mount_point)
             util.bindMount("/proc", "%s/proc" % dest_fs.mount_point)
+            util.bindMount("/run", "%s/run" % dest_fs.mount_point)
```

Comment:
Is it not worth using the ChrootMounts context manager for this section too?

Reply:
Done. Took a bit to test it.